dashboard/test/models/user_test.rb

```diff
@@ -177,12 +177,9 @@ class UserTest < ActiveSupport::TestCase
 
     user.age = 24
     assert_equal "21+", user.age
-    assert user.age
 
     user.save!
-
-    user = User.find(user.id)
-
+    user.reload
```

Comment:
is there a significant difference between `reload` and `find`? I don't see how this update changes much of anything